gulpfile.js

```diff
@@ -32,6 +32,8 @@ const paths = {
   packageJson: [
     'packages/core/parcel/package.json',
     'packages/utils/create-react-app/package.json',
+    'packages/dev/query/package.json',
+    'packages/dev/bundle-stats-cli/package.json',
```

Comment:
Hmm it works when I run `parcel-query` from src, but after running `yarn build` i get these lib/src errors

```
> parcel-query --help
/Users/bdo/parcel/packages/core/core/src/BundleGraph.js:3
import type {
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1178:20)
    at Module._compile (node:internal/modules/cjs/loader:1220:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32)
    at Module._load (node:internal/modules/cjs/loader:960:12)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/Users/bdo/parcel/packages/dev/query/lib/cli.js:70:20)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)

Node.js v18.17.0
```

Reply:
You need to handle these imports in `deep-imports.js` just like the ones for `AssetGraph` and so on

Speaking of that, should `@parcel/core` be a peerdependency of parcel-query?